app/models/user.rb

```diff
@@ -1,9 +1,12 @@
 class User < ApplicationRecord
+  validates :name, presence: true
+  validates :posts_counter, numericality: { greater_than_or_equal_to: 0 }
```

Comment:
- [ ] PostsCounter must be an integer greater than zero please fix that

Reply:
@DeliceLydia Then how do I create a New Post that has no comments or Likes?